app/features/modelling/line-plot/index.js

```diff
@@ -9,16 +9,56 @@ class LinePlotModel {
     fetch(`${src}?=`)
       .then(response => response.json())
       .then((data) => {
+        let datasets;
+        const { chartOptions, data: FEMdata, ...rest } = data;
+        if (FEMdata) {
+          datasets = FEMdata.map( (data, index) => {
+            const { seriesData, seriesOptions } = data
+            return {
+              data: seriesData,
+              label: `Series ${index + 1}`,
```

Comment:
Data series don’t have individual titles at the moment. It would be easy to add them to subjects. 
https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ScatterPlotViewer/README.md#series-data

Reply:
```suggestion
              label: seriesOptions?.label || `Series ${index + 1}`,
```